contracts/PetPark.sol

```diff
@@ -3,5 +3,106 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+    address owner;
+    enum AnimalType { None, Fish, Cat, Dog, Rabbit, Parrot }
+    enum Gender { Male, Female }
 
+    struct Person {
+        Gender gender;
+        uint age;
+        bool isRegistered;
+    }
+
+    mapping (uint => uint) animalCountByType;
+    mapping (address => Person) peopleByAddress;
+    mapping (address => uint) borrowedByAddress;
+
+    event Added (uint indexed animalType, uint indexed count);
+    event Borrowed (uint indexed animalType);
+    event Returned  (uint indexed animalType);
+
+
+    constructor() {
+        owner = msg.sender;
+    }
+
+    modifier onlyOwner() {
+        require(msg.sender == owner, "Not an owner");
+        _;
+    }
+
+    function animalCounts(uint _animalType) public view returns (uint) {
```

Comment:
Since solidity public mappings are automatically exposed as functions, you can just make `animalCounts` as a public mapping